CAM Platform/api-providers.md

```diff
@@ -611,30 +599,47 @@ To add an Azure subscription in Cloud Application Manager, you first have to upl
 }
 ```
 
-### GET /services/providers
```

Comment:
Please add above the specific parameters for any provider you add in the list of provider types at the beginning.

Reply:
Hi @robersantamaria I haven't quite understood the request for this change. it would be great if you add the comment on the changed file version to know the exact referenced line